Erp connector (#418)
akeneo/pim-api-docs: content/guides/erp-connection/step4-how-to-build-your-connector.md

# How to build your connector?

> Now that you have a good overview of your connector data flows and Akeneo PIM data and features, you are probably wondering how to build your connector...

Reviewer: ```suggestion
Now that you have a good overview of your connector data flows and Akeneo PIM data and features, you are probably wondering how to build your connector.
```

> Here are some of our recommendations to do this!

Reviewer: ```suggestion
Here are some of our recommendations!
```

> ## A "connection compliant" connector

Reviewer: ```suggestion
## A "Connection-compliant" connector
```

> The best way to interact with Julia's PIM is to develop a "[connection-compliant](https://help.akeneo.com/pim/serenity/articles/what-is-a-connection.html)" connector (based on our API).

Reviewer: ```suggestion
The best way to interact with Julia's PIM is to develop a "[Connection-compliant](https://help.akeneo.com/pim/serenity/articles/what-is-a-connection.html)" connector (based on our API).
```

Reviewer: Add a little explanation of what Connection is

:::tips
Why?

> Please don't hesitate to read our ["4 reasons why you should use our API"](https://api.akeneo.com/documentation/why-the-api.html#4-reasons-why-you-should-use-our-api) page!

Reviewer: ```suggestion
Please don't hesitate to read our ["4 reasons why you should use our API"](https://api.akeneo.com/documentation/why-the-api.html#4-reasons-why-you-should-use-our-api) page.
```

:::

## Follow the guide!

> All you need to know to interact with our PIM through our API can be find in our [dedicated tutorial](https://api.akeneo.com/getting-started/connect-the-pim-4x/welcome.html).

Reviewer: ```suggestion
All you need to know to interact with our PIM through our API can be found in our [dedicated tutorial](https://api.akeneo.com/getting-started/connect-the-pim-4x/welcome.html).
```

> ## Step by step...

Reviewer: ```suggestion
## Step by step
```

> **Our recommendation: always listen to your customer's needs first!**

Reviewer: ```suggestion
**Our recommendation: always listen to your customer's needs first.**
```

> We are aware that it is almost difficult to develop a 100% complete connector that fit all your customer needs immediately.

Reviewer: ```suggestion
We are aware that it is quite difficult to develop a 100% complete connector that fits all your customer's needs immediately.
```

> Furthermore, to build such a complex connector as an ERP connector in a single iteration would undoubtedly be complex and you risk missing important features.

Reviewer: ```suggestion
Once you are comfortable with your customer's needs, our recommendation is to follow the steps we describe and to iterate in order not to miss any important feature. Building an ERP connector is a complex task that requires a dedicated focus every step of the way.  
```

> **Our last advice: adopt an Agile approach!**

Reviewer: ```suggestion
**To conclude: adopt an Agile approach**
```

> * Interview Julia and ERP team about their needs

Reviewer: ```suggestion
* Interview Julia and ERP team about their needs
```
```suggestion
* Interview Julia and the ERP team to learn about their needs
```
